### PR TITLE
Includes the OAS Changes for the new service

### PR DESCRIPTION
**Pull request by jmsatheesh**

### CODE REVIEW

File: resources/public/api/conf/1.0/examples/create-organisation-response.json [hunk 1]
```diff
@@ -33,5 +33,6 @@
   "pensionSchemeAdministratorIdentifier":"A1234567",
   "eoriNumber":"GB123456789012",
   "crn": "12345678",
+  "personPresentingTheGoods": "GB123456789012",
```
Comment (peteslater-ee) [inline resources/public/api/conf/1.0/examples/create-organisation-response.json]: This should not be part of this PR - it fits better with https://github.com/hmrc/api-platform-test-user/pull/177

File: resources/public/api/conf/1.0/schemas/create-organisation-response.json [hunk 2]
```diff
@@ -149,6 +149,10 @@
       "type": "string",
       "description": "Company Reference Number"
     },
+    "personPresentingTheGoods": {
```
Comment (peteslater-ee) [inline resources/public/api/conf/1.0/schemas/create-organisation-response.json]: This should not be part of this PR - it fits better with https://github.com/hmrc/api-platform-test-user/pull/177

Comment (jmsatheesh) [reply resources/public/api/conf/1.0/schemas/create-organisation-response.json]: updated as per review comments

**Review by peteslater-ee — APPROVED** [review]
